packages/boost-libs/build

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
Mesos already includes a whole bunch of boost, what are we using which isn't bundled by mesos? Can we just use mesos bundling?

If we are going to do this outside of mesos, need to build it ones and configure mesos to use it.